xsuite/src/cli/cmd.test.ts

```diff
@@ -43,6 +43,8 @@ test("new-wallet --wallet wallet.json", async () => {
     "",
     chalk.bold.blue("Address:") + ` ${keystore.newSigner()}`,
     "",
+    chalk.bold.blue("Shard:") + ` ${keystore.getShard()}`,
```

Comment:
We could introduce a variable `const signer = keystore.newSigner();`.

Two lines above, we could use this variable `signer` instead of `keystore.newSigner()`.

And here, we could use `computeShard(signer.toTopHex())` instead of `keystore.getShard()`.

Finally, we could remove the `.getShard()` abstraction, which is also a bit confusing as a keystore doesn't belong to a shard, the first private key of the keystore does.